From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@TwoUnder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mausher). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve/disapprove with a comment here, thanks.

JennieHubbard commented:
#sign-off

SnehaGunda committed:
Adding compatibility level doc

SnehaGunda committed:
minor edit

tdykstra committed:
added c# attribute info, standardized format

itechedit committed:
edit pass: control-flow-lookup-activity

euangms committed:
New sample hashing a column value

Commit:
add rails tutorial

cephalin committed:
acrolinx

cwatson-cat committed:
Removed classic portal references from Automation topics.

TerryLanfear committed:
update to existing articles

TerryLanfear committed:
fixes

TerryLanfear committed:
fix

iainfoulds committed:
Initial draft

iainfoulds committed:
Edits and revisions

iainfoulds committed:
Adding next steps

iainfoulds committed:
Adding TOC entry

iainfoulds committed:
Acrolinx update

iainfoulds committed:
Adding line wraps

mdshihab committed:
(ForumsTeam) Typo

Commit:
updates to article

iainfoulds committed:
Updating intro

iainfoulds committed:
Edit and revisions

iainfoulds committed:
Resolving merge conflicts

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vmssaz

iainfoulds committed:
Updating Availability Zone overview and TOC to point to new doc

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

MicrosoftGuyJFlo committed:
Update .openpublishing.redirection.json

TerryLanfear committed:
final fixes

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-datacoll

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

pharring committed:
Document the TempFolder option for Cloud Services

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

pharring committed:
Merge remote-tracking branch 'upstream/master'

v-geberr committed:
how to debug

MicrosoftGuyJFlo committed:
Redirect for PR30772

v-geberr committed:
fix link

v-geberr committed:
removed internal subscription name

Commit:
automation: updates

Commit:
Add default message handler and bot.recognizer(recognizer) that was missing from template

v-geberr committed:
warning added

klaaslanghout committed:
The term network limit is not as common as 'throttle'.  Adding the searchable term to the article.

Commit:
corrected blade term

iainfoulds committed:
Incorporating feedback

Commit:
Fresheness - covert

SnehaGunda committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into CL

SnehaGunda committed:
Minor chnages

TerryLanfear committed:
final fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-datacoll

JasonWHowell committed:
Update TOC.yml

tfitzmac committed:
updated lock topic

bandersmsft committed:
policy - updated link

ktoliver committed:
Merge pull request #30703 from iainfoulds/vmssaz

VMSS Availability Zones

tdykstra committed:
acrolinx

jaredr80 committed:
Change advertised prefixes to match IP version

SnehaGunda committed:
More updates

ktoliver committed:
fix markdown in Important alert (add carets)

fix markdown in Important alert

ShawnJackson committed:
Update control-flow-lookup-activity.md

ktoliver committed:
Merge pull request #30781 from klaaslanghout/patch-9

The term network limit is not as common as 'throttle'.  Adding the se…

barbkess committed:
fix loading toc

ktoliver committed:
Merge pull request #30774 from pharring/master

Document TempFolder option

SnehaGunda committed:
Minor chnages

ShawnJackson committed:
Merge pull request #30418 from itechedit/control-flow-lookup-activity

edit pass: control-flow-lookup-activity

cwatson-cat committed:
Removed reference to classic portal in image alt text.

GitHubber17 committed:
Merge pull request #30788 from JasonWHowell/patch-3

MySQL: Adding PHP and Java tutorials to TOC

GitHubber17 committed:
Merge pull request #30790 from bandersmsft/master

policy - updated link

ktoliver committed:
Merge pull request #30789 from tfitzmac/0103lockupdate

updated lock topic

ktoliver committed:
Merge pull request #30793 from barbkess/livefyre

fix loading toc

cephalin committed:
review

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

SnehaGunda committed:
Heading update

SnehaGunda committed:
removing image

MicrosoftGuyJFlo committed:
Remove Azure Classic Portal

anrothMSFT committed:
adding known issues

Lisaco88 committed:
Merge pull request #29935 from SnehaGunda/CL

Adding compatibility level doc

Lisaco88 committed:
Merge pull request #30630 from TerryLanfear/sept-datacoll

update to existing articles

Lisaco88 committed:
Merge pull request #30780 from DeniseMak/1-3-nodefix

Add default message handler and bot.recognizer(recognizer) that was m…

Lisaco88 committed:
Merge pull request #30784 from cynthn/convert-unmanaged-to-managed-disks

Fresheness - convert

Lisaco88 committed:
Merge pull request #30791 from jaredr80/patch-1

Change advertised prefixes to match IP version

Lisaco88 committed:
Merge pull request #30794 from cwatson-cat/1-3-18-ClassicUpd-Automation

Removed reference to classic portal in image alt text.

Lisaco88 committed:
Merge pull request #30730 from mdshihab/patch-12

(ForumsTeam) Typo

Lisaco88 committed:
Merge pull request #30797 from SnehaGunda/master

Heading update

Lisaco88 committed:
Merge pull request #30801 from anrothMSFT/patch-6

adding known issues

Lisaco88 committed:
Merge pull request #30539 from cwatson-cat/12-21-ClassicClnUp-Automation

Removed classic portal references from Automation topics.

ggailey777 committed:
Reassign ownership of Mobile Apps files to Craig

Commit:
Edit saas-multitenantdb-schema-management.md (in sql-database/), as Acrolinx only 76.

Commit:
Acrolinx was 76, raising now to 82. I call Baloney on these scores, because I improved the article's clarity and readability by a lot, not just to 82. Too many bogus complaints by Acrolinx that I cannot or will not change just to please a mindless broad-brush algorithm. I benefit nicely from Acrolinx, but these numbers are ill-weighted.

euangms committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into euang420local

euangms committed:
Added a min max normalisation example

euangms committed:
Added custom module example

ggailey777 committed:
Added serverless and other cleanup

cephalin committed:
add to TOC

cephalin committed:
add link from QS

genlin committed:
update the steps

genlin committed:
change wording

nsoneji committed:
Updated ASR limit as per 2.1

Commit:
removing an entry from the domain joined TOC. It should not have been added here. It is still referenced by the other HDInsight TOCs.

genlin committed:
update the list

carolinacmoravia committed:
C9800: Inconsistent serial number

Hello, @spelluru,

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version. The include should be separated from the rest of the text.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

Commit:
automation : correct image

Commit:
updated images

YuriDio committed:
updating article

gsilva5 committed:
Updated and corrected limitations for Windows

moved some items out of limitations, removing the need for a service article

v-geberr committed:
updated key usage

GitHubber17 committed:
Merge pull request #30484 from euangms/master

New sample hashing a column value

ktoliver committed:
Merge pull request #30775 from MicrosoftGuyJFlo/MFAClassicRetireRedirect

Redirect for PR30772

ktoliver committed:
Merge pull request #30772 from MicrosoftGuyJFlo/MFAClassicRetire1

Remove Azure Classic Portal

gsilva5 committed:
Update create-vm-accelerated-networking-powershell.md

GitHubber17 committed:
Merge pull request #30806 from MightyPen/20180103-saas-edit-stragglers

Edit saas-multitenantdb-schema-management.md (in sql-database/).

GitHubber17 committed:
Merge pull request #30809 from ggailey777/patch-6

Added serverless and other cleanup

rjagiewich committed:
Merge pull request #30524 from cephalin/rails

add rails tutorial

GitHubber17 committed:
Update site-recovery-vmware-deployment-planner-analyze-report.md

GitHubber17 committed:
Merge pull request #30818 from Blackmist/freshness

removing an entry from the domain joined TOC. It should not have been…

rjagiewich committed:
Merge pull request #30381 from tdykstra/graph

Add C# compiled info to MS Graph binding ref doc

GitHubber17 committed:
Merge pull request #30819 from carolinacmoravia/patch-533

C9800: Inconsistent serial number

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #30815 from nsoneji/master

Updated ASR limit as per 2.1

GitHubber17 committed:
Merge pull request #30821 from georgewallace/autoqsupdate

automation : correct image

GitHubber17 committed:
Merge pull request #30822 from YuriDio/UpdatesJan2018_1

updating article

rjagiewich committed:
Merge pull request #30776 from v-geberr/0103-csharp-bot

0103 csharp bot

GitHubber17 committed:
Merge pull request #30803 from ggailey777/cleanup

Reassign ownership of Mobile Apps files to Craig

v-alje committed:
Merge pull request #30832 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #30778 from georgewallace/auto0103

automation : updates 1/3

GitHubber17 committed:
Update billing-add-change-azure-subscription-administrator.md

itechedit committed:
Merge pull request #30796 from MicrosoftGuyJFlo/MFAClassicRetire2

Mfa classic retire2

GitHubber17 committed:
Merge pull request #30813 from genlin/master73840

billing-add-change-azure-subscription-administrator.md

rjagiewich committed:
Merge pull request #30824 from gsilva5/patch-11

Updated and corrected limitations for Windows
